src/aspire/abinitio/commonline_c3_c4.py

```diff
@@ -680,16 +682,16 @@ def _J_sync_power_method(self, vijs):
         itr = 0
 
         # Power method iterations
+        logger.info("Initiating Power Method.")
```

Comment:
“power method”. Should we also specify what the power method is used for here to give some context?